.github/workflows/scripts/write-docker-plist.sh

```diff
@@ -23,6 +29,7 @@
 		</dict>
 	</dict>
 	<key>Version</key>
-	<string>59</string>
+	<string>${version}</string>
```

Comment:
The version can change depending on what Docker gets installed.
Unfortunately, I never figured out how to make brew pin a specific Docker version.